Beey.DataExchangeModel.Common/Orders/OrderInfoDto.cs

```diff
@@ -21,14 +21,19 @@ public class OrderInfoDto
     public ulong OrderNumber { get; set; }
     public uint Credit { get; set; }
     public decimal Amount { get; set; }
+    public uint Quantity { get; set; }
     public string? Currency { get; set; }
+    public string ProductName { get; set; }
     public string? Language { get; set; }
 
     [JsonConverter(typeof(JsonStringEnumConverter))]
     public PaymentStatus PaymentStatus { get; set; } = PaymentStatus.None;
     public string? PaymentPrCode { get; set; }
     public string? PaymentSrCode { get; set; }
     public string? PaymentResultText { get; set; }
+
+    public string? StripeCheckoutId { get; set; }
+    public string? StripeInvoiceId { get; set; }
```

Comment:
Tohle je takový zvláštní. Jak se to sem dostalo?

Reply:
Jo já jsem si asi znovu natáh hlavní branch a namíchaly se mi sem commity od Jitky.. a pak se objevily tady v tom pull requestu. Nějak s tím ještě neumím :)

Reply:
@kudlosh Každopádně nevím, co s tím mám dělat. Ten change není můj commit, ten je už v master. Přece jej nebudu revertovat, ne? To by naopak byla chyba. Po mergi se to sloučí, toto tam bude a bude to správně.